realm/realm-library/src/objectServer/java/io/realm/PermissionManager.java

```diff
@@ -518,6 +560,82 @@ public void cancel() {
         }
     }
 
+    private class MakeOfferAsyncTask extends AsyncTask<String> {
```

Comment:
@kneth It matches the public API method using it, but if you have any better suggestions I'm all ears.